kafka-ui-react-app/src/components/common/SmartTable/__tests__/SmartTable.spac.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render } from 'lib/testHelpers';
+import { screen } from '@testing-library/dom';
+import { Route } from 'react-router';
+import { clusterBrokersPath } from 'lib/paths';
+import fetchMock from 'fetch-mock';
+import { SmartTable } from 'components/common/SmartTable/SmartTable';
+import { TableState } from 'lib/hooks/useTableState';
+
+describe('SmartTable Component', () => {
+  afterEach(() => fetchMock.reset());
```

Comment:
why fetchMock here if we are not mocking any fetches and we don't need it since , we are giving it the data